src/index.js

```diff
@@ -37,13 +37,20 @@ function getCandidates ( resolved, extensions ) {
 	);
 }
 
+const HELPERS_ID = '\0commonjsHelpers';
+const HELPERS_NAME = '__commonjsHelpers';
```

Comment:
I realize this is no worse than it was, but this should be generated so as not to conflict with anything in the file. Could do a simple `indexOf` with an incrementing suffix rather than trying to be clever with scopes.